app/uk/gov/hmrc/cataloguefrontend/AuthController.scala

```diff
@@ -32,22 +32,21 @@ import views.html.sign_in
 import scala.concurrent.Future
 
 @Singleton
-class AuthController @Inject()(val messagesApi: MessagesApi, authService: AuthService, configuration: Configuration)
-    extends FrontendController
-    with I18nSupport {
+class AuthController @Inject()(
+  authService: AuthService,
+  configuration: Configuration,
+  mcc: MessagesControllerComponents
+) extends MessagesAbstractController(mcc) with FrontendController {
 
   import AuthController.signinForm
 
-  private val selfServiceUrl = {
-    val key = "self-service-url"
-    configuration.getString(key).getOrElse(throw new Exception(s"Expected to find $key"))
-  }
+  private[this] val selfServiceUrl = configuration.get[String]("self-service-url")
 
-  val showSignInPage = Action { implicit request =>
+  val showSignInPage: Action[AnyContent] = Action { implicit request =>
```

Comment:
We usually don't add `: Action[AnyContent]`, it's not wrong defnitely and feel free to make a decision to keep it or not